virl2_client/models/node.py

```diff
@@ -495,6 +499,18 @@ def resource_pool(self) -> str:
         self.lab.sync_operational_if_outdated()
         return self._resource_pool
 
+    @property
+    def locked_compute_id(self) -> str | None:
+        """Return the ID of the compute this node is locked to."""
+        self.lab.sync_operational_if_outdated()
```

Comment:
I think this should go to `self.node.lab.sync_topology_if_outdated()` as it cannot be changed why a node is running (`compute_id` is the operational property, but that's already covered).

Reply:
It can't. It's not *in* the topology. The only sync function that actually retrieves node data directly from the `{lab}/nodes/{id}/` endpoint (the only place which contains `locked_compute_id`) is `Node.sync_operational` (as a side effect of getting operational data) so I just hijacked that.